ember-string/tsconfig.json

```diff
@@ -0,0 +1,42 @@
+{
+  "extends": "@tsconfig/ember/tsconfig.json",
+  "include": [
+    "src/**/*",
+    "unpublished-development-types/**/*"
+  ],
+  "compilerOptions": {
+    "allowJs": true,
+    "declarationDir": "declarations",
+    "noEmit": false,
```

Comment:
these 3 settings here were required to generate declarations with `tsc --declaration`.

I need to look in to if it makes sense to put these in the addon-blueprint, or if we should just "fork" `@tsconfig/ember` at this point (still in tsconfig/bases tho) maybe with a v2 app the delta can start shrinking again